test/selenium/configurations.rb

```diff
@@ -42,6 +42,7 @@ def common_configuration_for_firefox_extension
   xpi_filename = "@" + json['name'] + "-" + json['version'] + ".xpi"
   # Load the Firefox driver with the extension installed
   @profile = Selenium::WebDriver::Firefox::Profile.new
+  @profile["extensions.privly.integration_test"] = "true"
```

Comment:
[Extension background script](https://github.com/privly/privly-jetpack/pull/11/files#diff-0d32a96b97e2c5e320d8df9cd42188e1R26) and [Test runner](https://github.com/privly/privly-jetpack/pull/11/files#diff-eb74954aff54d42f3dfa90cb1410ce87R54).
Injects the test content script(used for testing context messenger) only when the extension integration tests are run.